lib/vendor/de77/BMP.php

```diff
@@ -46,7 +46,7 @@ public static function imagebmp(&$img, $filename = false)
 		$hei     = imagesy($img);
 		$wid_pad = str_pad('', $wid % 4, "\0");
 
-		$size = 54 + ($wid + $wid_pad) * $hei * 3; //fixed
+		$size = 54 + ($wid + strlen($wid_pad)) * $hei * 3; //fixed

```

Comment:
The formula in the https://github.com/smottt/WideImage/pull/18#issuecomment-757241933 is correct, provided we also fix `$wid_pad`. Otherwise this will calculate the same incorrect padding `3` and multiply it by `3` gaining even worse total padding of `9`.